apps/dashboards/tests/test_api.py

```diff
@@ -92,3 +96,33 @@ def test_delete(self):
         url = reverse("dashboard-detail", kwargs={"pk": fake_data.id})
         response = self.client.delete(url)
         assert response.status_code == 204
+
+
+class Test_Dashboard_and_VisualizationViewSet_non_Staff_Request(
```

Comment:
Name is too long, move these tests to their respective viewset test cases. See below for more details.

Reply:
Fixed